needs a little more space to handle pthread mutex_t with some ABIs

http://code.google.com/p/mozc/issues/detail?id=177

--- a/base/mutex.h
+++ b/base/mutex.h
@@ -52,8 +52,8 @@
 #define MOZC_RW_MUTEX_PTR_ARRAYSIZE 32
 #else
 // Currently following sizes seem to be enough to support all other platforms.
-#define MOZC_MUTEX_PTR_ARRAYSIZE 6
-#define MOZC_RW_MUTEX_PTR_ARRAYSIZE 10
+#define MOZC_MUTEX_PTR_ARRAYSIZE 8
+#define MOZC_RW_MUTEX_PTR_ARRAYSIZE 12
 #endif
 
 class LOCKABLE Mutex {
